### PR TITLE
Cleanup Helm chart code

### PR DESCRIPTION
**Pull request by Donnerbart**
Use JDK 12+ features and fix simple warnings in the code base.

### CODE REVIEW

File: tests-hivemq-operator/src/integrationTest/java/com/hivemq/helmcharts/testcontainer/OperatorHelmChartContainer.java [hunk 1]
```diff
@@ -147,10 +146,10 @@ public void upgradeLocalChart(final @NotNull String chartName, final @NotNull St
                 valuesFilePath);
 
         if (!execDeploy.getStderr().isEmpty()) {
-            // Shows also warnings
-            System.err.println(execDeploy.getStderr());
+            // shows also warnings
+            LOG.warn(execDeploy.getStderr());
         }
-        assertFalse(execDeploy.getStdout().isEmpty(), execDeploy.getStderr());
+        assertThat(execDeploy.getStdout().isEmpty()).describedAs(execDeploy::getStderr).isFalse();
```
Comment (Donnerbart) [inline tests-hivemq-operator/src/integrationTest/java/com/hivemq/helmcharts/testcontainer/OperatorHelmChartContainer.java:152]:
```suggestion
        assertThat(execDeploy.getStdout()).describedAs(execDeploy::getStderr).isNotEmpty();
```

File: tests-hivemq-platform-operator/src/integrationTest/java/com/hivemq/helmcharts/AbstractHelmChartIT.java [hunk 2]
```diff
@@ -24,8 +24,8 @@ public abstract class AbstractHelmChartIT {
     protected final @NotNull HelmChartContainer helmChartContainer = new HelmChartContainer().withNetwork(network);
 
     protected final @NotNull String namespace = K8sUtil.getNamespaceName(this.getClass());
-    protected final @NotNull String platformReleaseName = getPlatformReleaseName();
-    protected final @NotNull String operatorReleaseName = getOperatorReleaseName();
+    protected final @NotNull String platformReleaseName = "test-hivemq-platform";
```
Comment (Donnerbart) [inline tests-hivemq-platform-operator/src/integrationTest/java/com/hivemq/helmcharts/AbstractHelmChartIT.java:27]: Convert to constants

File: tests-hivemq-platform-operator/src/integrationTest/java/com/hivemq/helmcharts/single/HelmInitContainerIT.java [hunk 3]
```diff
@@ -34,11 +34,11 @@ void withAdditionalInitContainer_hivemqRunningWithVolumeMounts() throws Exceptio
 
         await().atMost(5, TimeUnit.MINUTES).untilAsserted(() -> {
             final var statefulSet =
-                    client.apps().statefulSets().inNamespace(namespace).withName(getPlatformReleaseName()).get();
+                    client.apps().statefulSets().inNamespace(namespace).withName("test-hivemq-platform").get();
```
Comment (afalhambra-hivemq) [inline tests-hivemq-platform-operator/src/integrationTest/java/com/hivemq/helmcharts/single/HelmInitContainerIT.java:37]:
```suggestion
                    client.apps().statefulSets().inNamespace(namespace).withName(platformReleaseName).get();
```

File: tests-hivemq-operator/src/integrationTest/java/com/hivemq/helmcharts/testcontainer/OperatorHelmChartContainer.java (hunk 1 above)
Comment (afalhambra-hivemq) [reply tests-hivemq-operator/src/integrationTest/java/com/hivemq/helmcharts/testcontainer/OperatorHelmChartContainer.java:152]: I guess, you forgot to commit your suggestion here? :)

**Review by afalhambra-hivemq — APPROVED** [review]
pretty nice clean-up!

File: tests-hivemq-operator/src/integrationTest/java/com/hivemq/helmcharts/testcontainer/OperatorHelmChartContainer.java (hunk 1 above)
Comment (Donnerbart) [reply tests-hivemq-operator/src/integrationTest/java/com/hivemq/helmcharts/testcontainer/OperatorHelmChartContainer.java:152]: I used it just as a reminder, when scrolling through this after work. I prefer to fix this in IDEA and rebase it into the original commit.

**Pull request merged**